ml-algorithms/build.gradle

```diff
@@ -20,6 +20,7 @@ dependencies {
     implementation group: 'org.tribuo', name: 'tribuo-clustering-kmeans', version: '4.2.1'
     implementation group: 'org.tribuo', name: 'tribuo-regression-sgd', version: '4.2.1'
     implementation group: 'org.tribuo', name: 'tribuo-anomaly-libsvm', version: '4.2.1'
+    implementation group: 'org.tribuo', name: 'tribuo-classification-experiments', version: '4.2.1'
```

Comment:
This is going to pull in a bunch of Tribuo, including the XGBoost native library, which you might not want. If you just want the SGD logistic regression then you can pull in `tribuo-classification-sgd`.

Reply:
Thanks for the comment. This is a good point. :)
I wanted to use this congregated one to avoid pulling multiple different packages in the first place, but it seems ends up only using "tribuo-classification-sgd".